code/temperature-converter/src/main/java/ru/unn/agile/TemperatureConverter/model/CelsiusTemperature.java

```diff
@@ -0,0 +1,17 @@
+package ru.unn.agile.TemperatureConverter.model;
```

Comment:
use low-case for package names, please

Reply:
I think the names of the packages should be consistent in the project.

In the korniakov-kirill-lab1 module, the package name is as follows: 
`package ru.unn.agile.ComplexNumber.model;`

Reply:
Currently - yes. But the package name in ComplexNumber will be fixed. So could you rename you package to low-case? Thx.

Reply:
Ok, done.